src/main/scala/is/hail/methods/LogisticRegression.scala

```diff
@@ -36,13 +36,13 @@ object LogisticRegression {
     if (d < 1)
       fatal(s"$n samples and $k ${plural(k, "covariate")} including intercept implies $d degrees of freedom.")
 
-    info(s"Running logreg on $n samples with $k ${ plural(k, "covariate") } including intercept...")
+    info(s"Running $test logreg on $n samples with $k ${ plural(k, "covariate") } including intercept...")
 
     val nullModel = new LogisticRegressionModel(cov, y)
-    val nullFit = nullModel.nullFit(nullModel.bInterceptOnly())
+    val nullFit = nullModel.fit()
```

Comment:
fit always defines them (they are computed in each iteration anyway, even if not test types need them), fitFirth never does.

In Firth, I use the null model from fit to initialize the the per-variant null model for Firth.  The null model for Firth then initializes the full model for Firth (neither use score or Fisher from the first null model though).